src/index.html

```diff
@@ -11,7 +11,197 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="elements">
+          <label>
+            Surname:
+            <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            >
```

Comment:
```suggestion
            <input
              type="text"
              autocomplete="off"
              name="surname"
            >
```
checklist p.3
fix code style all over the page